src/main/java/jenkins/plugins/logstash/LogstashConsoleLogFilter.java

```diff
@@ -3,64 +3,110 @@
 import java.io.IOException;
 import java.io.OutputStream;
 import java.io.Serializable;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 import hudson.Extension;
 import hudson.console.ConsoleLogFilter;
-import hudson.model.AbstractBuild;
 import hudson.model.AbstractProject;
 import hudson.model.Run;
 
 @Extension(ordinal = 1000)
 public class LogstashConsoleLogFilter extends ConsoleLogFilter implements Serializable
 {
 
+  private static Logger LOGGER = Logger.getLogger(LogstashConsoleLogFilter.class.getName());
   private transient Run<?, ?> run;
-  public LogstashConsoleLogFilter() {};
+
+  public LogstashConsoleLogFilter()
+  {};
 
   public LogstashConsoleLogFilter(Run<?, ?> run)
   {
     this.run = run;
   }
+
   private static final long serialVersionUID = 1L;
 
   @Override
   public OutputStream decorateLogger(Run build, OutputStream logger) throws IOException, InterruptedException
   {
-    if (build != null && build instanceof AbstractBuild<?, ?>)
+    /*
+     * Currently pipeline is not supporting global ConsoleLogFilters. So even when we have it enabled globally
```

Comment:
I think this would be valid javadoc describing how the method works.
Javadoc would also be more visible in IDEs etc.